<?xml version="1.0"?>
<launch>
    <node pkg="twist_controller" type="dbw_node" name="dbw_node">
        <param name="vehicle_mass" value="1736.35" />
        <param name="fuel_capacity" value="13.5" />
        <param name="brake_deadband" value="100." />
        <param name="decel_limit" value="-1." />
        <param name="accel_limit" value="1." />
        <param name="wheel_radius" value="0.2413" />
        <param name="wheel_base" value="2.8498" />
        <param name="steer_ratio" value="14.8" />
        <param name="max_lat_accel" value="3." />
        <param name="max_steer_angle" value="8." />
        <param name="kp_vel" value ="2."/>
        <param name="kp_throttle" value ="0.5"/>
        <param name="ki_throttle" value ="0.0000"/>
        <param name="min_speed" value ="1."/>

        <remap from="steering_cmd" to="/vehicle/steering_cmd" />
        <remap from="throttle_cmd" to="/vehicle/throttle_cmd" />
        <remap from="brake_cmd" to="/vehicle/brake_cmd" />
        <remap from="dbw_enabled" to="/vehicle/dbw_enabled" />
    </node>
</launch>
